code/game/machinery/cloning.dm

```diff
@@ -126,6 +126,15 @@
 	if (is_operational() && (!isnull(occupant)) && (occupant.stat != DEAD))
 		to_chat(user, "Current clone cycle is [round(get_completion())]% complete.")
 
+/obj/machinery/clonepod/return_air()
+	// We want to simulate the clone not being in contact with
+	// the atmosphere, so we'll put them in a constant pressure
+	// nitrogen. They'll breathe through the chemicals we pump into them.
+	var/global/datum/gas_mixture/immutable/cloner/GM //global so that there's only one instance made for all cloning pods

```

Comment:
You're the one arguing over it, we've long since accepted the use of ```static```, we know it's not going anywhere because Lummox has confirmed it (and byond itself is very very very backwards compatible (to it's detriment)), and MSO even has an FR up to replace ```global``` entirely with ```static```

```global``` as a name makes NO sense for what we're using it for, ```static``` is literally perfect, which iirc is what I told you last time.

Which reminds me, @MrStonedOne, bump your FR http://www.byond.com/forum/?post=2019425